bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/OpenAPIUtils.java

```diff
@@ -66,28 +78,42 @@ public static Request requestBuilder(HttpClient httpClient, NanoleafControllerCo
         }
         URI requestURI;
         try {
-            requestURI = new URI(HttpScheme.HTTP.asString(), null, address, port, path, null, null);
+            requestURI = new URI(HttpScheme.HTTP.asString(), null, address, port, path, query, null);
         } catch (URISyntaxException use) {
-            logger.warn("URI could not be parsed with path {}", path);
+            LOGGER.warn("URI could not be parsed with path {}", path);
             throw new NanoleafException("Wrong URI format for API request");
         }
-        return httpClient.newRequest(requestURI).method(method);
+        return requestURI;
     }
 
     public static ContentResponse sendOpenAPIRequest(Request request)
             throws NanoleafException, NanoleafUnauthorizedException {
         try {
+            LOGGER.trace("Sending Request {} {}",request.getURI(), request.getQuery() == null ? "no query parameters": request.getQuery());
+            LOGGER.trace("Request method:{} uri:{} params{}\n", request.getMethod(), request.getURI(), request.getParams());
+            if (request.getContent()!=null) {
```

Comment:
Formatting is non-standard

Reply:
Do you refer to trace/log formatting or code formatting here? 
If the latter, I applied spotless and please mark as resolved.

Reply:
Indeed